### PR TITLE
fix(import-dir): Ignore file extensions | Ignore .test files

### PR DESCRIPTION
**Pull request by dac09**
**What does it do?**
Fixes redwood dir importer babel plugin to:
- remove extensions
- not import test files

[Community Discussion Reference](https://community.redwoodjs.com/t/typescript-prod-deployment/740/23)

### CODE REVIEW

**Comment by peterp** [comment]
WOW 🤩

File: packages/core/src/babel-plugin-redwood-import-dir.ts [hunk 1]
```diff
@@ -47,17 +47,30 @@ export default function ({ types: t }: { types: typeof types }): PluginObj {
...
           const fileName = path.basename(filePath).split('.')[0]
           // + import * as <importName>_<fileName> from <filePath>
+
+          const parsedPath = path.parse(filePath)
+
+          // Do it this way to allow double dots e.g. service/payment/payment.utils.ts
+          const filePathWithoutExtension = path.join(
+            parsedPath.dir,
+            parsedPath.name
+          )
```
Comment (peterp) [inline packages/core/src/babel-plugin-redwood-import-dir.ts:64]: node imports paths are normalized across operating systems, so I don't think we need to use `path.join()` here, instead `.join('/')` should be enough.

Comment (dac09) [reply packages/core/src/babel-plugin-redwood-import-dir.ts:64]: 👍 Change coming up

**Comment by dac09** [comment]
It might be a good idea to update the default scaffold to use the dir-import instead of `importAll` too - not sure where this is though!

**Comment by peterp** [comment]
> It might be a good idea to update the default scaffold to use the dir-import instead of importAll too 

You're right, it's probably time, and then we can remove importAll.macro from the api side!

That needs to be done over here: https://github.com/redwoodjs/create-redwood-app/blob/main/api/src/functions/graphql.js

**Pull request merged**